### ISSUE TITLE
Gutenberg: v11.0.x plugin upgrade

### ISSUE THREAD

**Issue by fullofcaffeine** [Automattic/wp-calypso#54180]
Previous upgrade: https://github.com/Automattic/wp-calypso/issues/53725.
Release notes: [v11.0.0-rc.1](https://github.com/WordPress/gutenberg/releases/tag/v11.0.0-rc.1),  [v11.0.0](https://github.com/WordPress/gutenberg/releases/tag/v11.0.0).

- [x] Install and activate
  - [x] Copy & install v11.0.0-rc.1 (D63527-code, D63537-code)
  - [x] Activate v11.0.0-rc.1 on edge (D63539-code)
  - [x] Copy & install v11.0.0 (D63799-code, D63805-code)
  - [x] Activate v11.0.0 on edge (D63808-code)
  - [x] Activate v11.0.0 on production (D63540-code)
  - [x] Request AT upgrade (p9o2xV-1wd-p2)
- [x] Testing
  - [x] Desktop viewport E2E 
  - [x] Mobile viewport E2E
  - [x] WPCOM (D63540-code)
- [x] Publish internal announcements
- [ ] Clean up unused releases
- [x] Add log entry to the Gutenberg Upgrade Log [pcoGjb-g-p2]
- [x] Open a new issue for the next upgrade, transfer remaining tasks, close this issue.

### Other issues 🐛

- [x] `Site Logo` block hangs in the loading state
    - Reported here: p1625693489206000/1625262437.120500-slack-C7YPUHBB2
    - Fixed in this diff by @creativecoder: D59822-code
    - TL;DR: Chain of events -- some needed attributes were not present in the API response (due to the different API setup in WPCOM, namely the `public-api`), and this was causing an expression in the block js to evaluate to `undefined` which was would then put it in the loading state, forever.
    

#### Transfered from previous release(s)

---
cc @inaikem @simison

**Comment by simison** [comment]
This should fix https://github.com/Automattic/wp-calypso/issues/53401

**Comment by fullofcaffeine** [comment]
Next one: https://github.com/Automattic/wp-calypso/issues/54393.
